detekt/detekt.bzl

```diff
@@ -4,35 +4,43 @@ Detekt is a static analysis tool for Kotlin: https://github.com/arturbosch/detek
 """
 
 def _impl(ctx):
+    action_inputs = [] + ctx.files.srcs
+
+    # See Detekt CLI documentation: https://arturbosch.github.io/detekt/cli.html
     # TODO: Allow customizing JVM options.
-    arguments = [
+    action_arguments = [
```

Comment:
Did you consider an [`Args`](https://docs.bazel.build/versions/master/skylark/lib/Args.html) object?

```python
action_arguments = ctx.actions.args()
action_arguments.add_all([
    "-Xms16m",
    "-Xmx128m",
    "-jar",
    ctx.file._detekt_cli_jar.path,
])
```

Reply:
ah nice